docs/reference/npm-scripts-and-nx.md

```diff
@@ -0,0 +1,120 @@
+---
+title: NPM / NX Guidelines
+---
+
```

Comment:
I think it's good to have an "Up to date as of ______" at the top of each page, since a lot of our stuff can get stale